packages/playground/website/src/github/github-export-form/form.tsx

```diff
@@ -430,10 +430,10 @@ export default function GitHubExportForm({
 	if (pushResult) {
 		return (
 			<form id="export-playground-form" onSubmit={handleSubmit}>
-				<h2 tabIndex={0} style={{ marginTop: 0, textAlign: 'center' }}>
+				<h3>
```

Comment:
Why did these change from `<h2>` to `<h3>`? It doesn't look like there are any other `<h2>`'s in the modal. The modal title appears to be an `<h1>`.

Reply:
Good catch! This is a leftover after merging other branches. The old modal component (the custom one) had title as h2. Fixed.